bessctl/conf/testing/run_module_tests.bess

```diff
@@ -106,20 +106,46 @@ def crash_test(module, num_input_ports, num_output_ports):
     time.sleep(5)
     bess.pause_all()
 
-# Can't just call exec() in the middle of main loop because
-# variables will leak into main script scope
 def load_test(filename):
-    CRASH_TEST_INPUTS = []
-    OUTPUT_TEST_INPUTS = []
-    CUSTOM_TEST_FUNCTIONS = []
+    """
+    Load a test file.  It should modify the (global to it) variables
+    CRASH_TEST_INPUTS, OUTPUT_TEST_INPUTS, and CUSTOM_TEST_FUNCTIONS
+    using .append() to add its tests.  The format of each of these is
+    a bit different; see the wiki at
+    https://github.com/NetSys/bess/wiki/Python-Script-Namespaces
```

Comment:
Oops, yes, I moved the page (before actually creating it) when I made it more general. Should I re-push the pull request with corrected web address, or just fix it later?